src/main/java/com/kuzmins/service/TicketService.java

```diff
@@ -79,93 +79,43 @@ public List<Ticket> returnTicketsBySector(Sector sector) {
-
-        //put(Ticket ticket, int index) + deleteByIndex(int index)
         Ticket ticket7 = new Ticket("Tabernacle", "123",
-                Instant.parse("2024-12-12T19:00:00.000Z"), true, Sector.C,
+                Instant.parse("2024-12-12T19:00:00.000Z"), true, Sector.C, TicketType.YEAR,
                 0.000, new BigDecimal("50.00"));
 
-        customArrayList.put(ticket7, 0);
-        System.out.println(customArrayList.getByIndex(0));
-        customArrayList.deleteByIndex(0);
-        System.out.println(customArrayList.getByIndex(0));
-
+        User kate = new User("kate");
+        User matt = new User("matt");
 
-        Ticket[] tickets = new Ticket[3];
-        List<Ticket> arrayList = service.returnTicketsBySector(Sector.B);
-        System.arraycopy(arrayList.toArray(), 0, tickets, 0, tickets.length);
+        ticketDAO.saveTicket(ticket10);
+        userDAO.saveUser(kate);
 
-        //add(Ticket [] tickets)
-        customArrayList.add(tickets);
-        for (int i = 0; i < customArrayList.size(); i++) {
-            System.out.println(customArrayList.getByIndex(i));
-        }
+        ticketDAO.saveTicket(ticket9, kate);
+        ticketDAO.saveTicket(ticket8, kate);
 
-        //add(Ticket [] tickets, int index)
-        customArrayList.add(tickets, 0);
-        for (int i = 0; i < customArrayList.size(); i++) {
-            System.out.println(customArrayList.getByIndex(i));
-        }
+        userDAO.saveUser(matt);
+        ticketDAO.saveTicket(ticket7, matt);
 
-        //put(Ticket [] tickets, int index)
-        customArrayList.put(tickets, 9);
-        for (int i = 0; i < customArrayList.size(); i++) {
-            System.out.println(customArrayList.getByIndex(i));
-        }
+        Ticket ticket = ticketDAO.fetchTicketById(ticket7.getId());
 
-/* CustomHashSet */
-
-        CustomHashSet<Ticket> customHashSet = new CustomHashSet<>();
-
-        Ticket ticket1 = new Ticket("Tabernacle", "123",
-                Instant.parse("2024-12-12T19:00:00.000Z"), false, Sector.B,
-                4.547, new BigDecimal("70.00"));
-        Ticket ticket2 = new Ticket("Radio City", "007",
-                Instant.parse("2024-11-29T19:00:00.000Z"), true, Sector.A,
-                1.12, new BigDecimal("50.00"));
-        Ticket ticket3 = new Ticket("Red Rocks", "987",
-                Instant.parse("2024-10-26T20:00:00.000Z"), true, Sector.C,
-                3.50, new BigDecimal("50.00"));
-
-        //put (T element)
-        customHashSet.put(ticket1);
-        customHashSet.put(ticket2);
-        customHashSet.put(ticket3);
-
-        //keep objects uniqueness
-        customHashSet.put(ticket3);
-
-        //iterate
-        for (Ticket ticket : customHashSet) {
-            System.out.println(ticket);
-        }
+        List<Ticket> ticketList = ticketDAO.fetchTicketsByUserId(kate.getId());
 
-        //delete(T element)
-        customHashSet.delete(ticket1);
+        User user3 = userDAO.fetchUserById(matt.getId());
```

Comment:
add one function: 'Fetch Tickets by ID and user ID' in one command

Reply:
I'm not sure that I understand the comment)
Do you mean to write a method fetchTicketsByUserId(UUID userId) call just to test, how does it work?

Or do I need to add a new method into ticketDAO?

Reply:
yes, it is about new complex method: 
```java
public Ticket fetchTicketByIdAndUserId(UUID id, UUID userId) {
...
...prepareStatement("SELECT * FROM ticket_info JOIN user_info ON user_info.id = ticket_info.user_id WHERE ticket_info.id=? and user_info.id=?")
...
```